### DISCUSSION TITLE
slimbookbattery for kinetic

### DISCUSSION THREAD

**Discussion by cesarcoatl** [Slimbook-Team/slimbookbattery#92] [Q&A] [answered]
Hello there,

I wanted to install slimbookbattery on Ubuntu 22.10 (kinetic), but `kinetic` is not found on your PPA, so it is ignored.

Should I wait for you to add `kinetic`?

Should I edit `/etc/apt/sources.list.d/slimbook-ubuntu-slimbook-kinetic.list` and switch `kinetic` for `jammy`?

Thanks

**Comment by slimbook** [comment, marked as answer]
Hi, I've copied last version package for kinetic [here](https://launchpad.net/~slimbook/+archive/ubuntu/slimbook?field.series_filter=kinetic) ;)
